app/src/main/java/com/github/swent/echo/data/room/RoomLocalDataSource.kt

```diff
@@ -97,7 +131,6 @@ class RoomLocalDataSource @Inject constructor(db: AppDatabase) : LocalDataSource
             events.flatMap { event -> event.tags.map { EventTagCrossRef(event.eventId, it.tagId) } }
 
         eventDao.deleteAllEventTagCrossRefsForEvents(events.map { it.eventId })
-        associationDao.insertAll(associations.map { AssociationRoom(it) })
```

Comment:
The variable `associations` is now unused and could be deleted.

Reply:
you're right, I'll remove that :)